router/keeper/keeper.go

```diff
@@ -30,9 +34,15 @@ type Keeper struct {
 
 var (
 	// Timeout height following IBC defaults
-	DefaultTransferPacketTimeoutHeight = clienttypes.MustParseHeight(transfertypes.DefaultRelativePacketTimeoutHeight)
+	DefaultTransferPacketTimeoutHeight = clienttypes.Height{
```

Comment:
Is there a reason for these defaults or is this just temporary for testing?

Reply:
These are logical defaults, but the chain has the ability to override, and the packet also has the ability to override.

Reply:
I meant: why is this preferred to the previous values of `clienttypes.MustParseHeight(transfertypes.DefaultRelativePacketTimeoutHeight)`. I don't have an opinion here, just curious about the reason behind the change